src/utility/Constants.js

```diff
@@ -21,3 +21,52 @@ export const SOCIAL_LINKS = Object.freeze({
   TW: 'https://www.twitter.com/nwplusubc',
   WEBSITE: 'https://www.nwplus.io',
 })
+
+export const applicantStatus = Object.freeze({
+  attending: 'attending',
+  new: 'new',
+  applied: 'applied',
+  accepted: 'accepted',
+  inProgress: 'inProgress',
```

Comment:
Also on another note, can we add waitlisted, rejected here too? the dashboard (@andykwan1124) has views for waitlist and rejected applicants too

Reply:
I think I'll do proper redirecting in a separate PR and update all the status